### DISCUSSION TITLE
数字排序问题

### DISCUSSION THREAD

**Discussion by monsterxcn** [AlistGo/alist#2148] [Q&A]
最近下了点好康的图片放在 OneDrive 里，想用 AList 在线看，但是似乎图片的排序有些难受，请问有什么办法能让图片按 1, 2, 3, ..., 10, 11, 12, ..., 100, 101 这样正常的顺序排呢？

现在的排序是这样的（创建存储的时候选的是按文件名升序排序）：

![image](https://user-images.githubusercontent.com/22407052/198817577-dde4527e-a0e9-43a4-9ff8-c316b933ca37.png)

**Comment by ywysuibian** [comment]
【我用的阿里云盘】创建存储选择排序后，页面默认加载出来也是这样错的排序，但是点一下名称它又能排对了，希望修复
![image](https://github.com/alist-org/alist/assets/12655790/54435627-af1d-433c-a505-17fa35211d07)

**Comment by zzzhoin** [comment]
我也是这个问题  有人提issue了吗  讨论这边的内容会不会看啊
